app/controllers/replies_controller.rb

```diff
@@ -13,7 +13,7 @@ def search
     @page_title = 'Search Replies'
     use_javascript('search')
 
-    @post = Post.find_by(id: params[:post_id]) if params[:post_id].present?
+    @post = Post.find_by(id: params[:post_id]).includes([:audits]) if params[:post_id].present?
```

Comment:
Hm, I'm concerned about this because I think we're not supposed to *use* audits on the reply search page (we set it to [] on purpose so we don't do the query). This should more likely be fixed by finding if/where it's loading them and telling it to knock it off; we don't care in the context of search results.

Reply:
Ah, okay. That seems easy enough.